config/rbac/role.yaml

```diff
@@ -669,6 +673,15 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
Is there a corresponding Kubebuilder tag somewhere?

Reply:
for features.opendatahub.io? 
https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/controllers/dscinitialization/dscinitialization_controller.go#L71

Reply:
Yes. I see it's likely a leftover from #801.